client/src/routes/fight/FightSideProposition.svelte

```diff
@@ -0,0 +1,45 @@
+<script>
+    import Button, {Label} from "@smui/button";
+
+    function handleClick(type){
+        console.log("clicked!")
+    }
+</script>
+
+<div class="sideDiv">
+    <p>Proponowane zdarzenia od sędziów bocznych:</p>
+    <Button class="redButton" disabled=true on:click={() => handleClick("red")}>
+    </Button>
+    <Button class="blueButton" on:click={() => handleClick("blue")}>
+        <Label>[informacje od niebieskiego]</Label>
+    </Button>
+</div>
+
+<style>
+    .sideDiv{
+        display: flex;
+        align-items: center;
+        flex-direction: column;
+    }
+
+    p{
+        color: var(--blue-fighter);
+    }
+
+    * :global(.blueButton){
+        background-color: var(--blue-fighter);
+    }
+
+    * :global(.redButton){
+        background-color: var(--red-fighter);
+    }
+    * :global(Button){
+        width: 85vw;
```

Comment:
`vw` should work fine, but I think it would be better to use `%`, as the component will then scale in respect to the parent component, not the whole viewport. For example, we can fix parent component's width to, for example, 720px for larger viewport's widths, and then button will scale accordingly.